04-simulate-an-election/election.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+
+def elect_in_region(election_result, candidate_chances):
+    if election_result <= candidate_chances:
+        return 'a'
+    else:
+        return 'b'
+
+def simulate_a_single_election():
+    candidate_a_chances = {
+        "region1": 87,
+        "region2": 65,
+        "region3": 17
+    }
+
+    candidate_a_elected_count = 0
+    candidate_b_elected_count = 0
+
+    for region in candidate_a_chances:
+        elected = elect_in_region(election_result=int(random() * 100), candidate_chances=candidate_a_chances[region])
+
+        if elected == "a":
+            candidate_a_elected_count += 1
+        else:
+            candidate_b_elected_count += 1
+
+    if candidate_a_elected_count > candidate_b_elected_count:
+        return 'a'
+    else:
+        return 'b'
+
+def simulate_elections(elections_count):
+    candidate_a_elections_count = 0
+    candidate_b_elections_count = 0
+
+    for i in range(0, elections_count):
```

Comment:
When you want to produce an array of values between 0 and X, you don't have to specify the 0, this will suffice: `range(X)`.